redlite/model/hf_model.py

```diff
@@ -39,6 +40,18 @@ def __init__(self, hf_name: str, device: str | None = None, token=None, max_leng
             .eval()
         )
 
+        else:    
```

Comment:
This code is not robust, as it apparently assumes that device value is string. How about `"tpu"` ? or `torch.device("cpu")`

Reply:
Please click on 3 dots to view the file. I have already handle torch.device("cpu"), but not "tpu".

Reply:
Please review commit [feb7df4](https://github.com/innodatalabs/redlite/pull/9/commits/feb7df4b7a63b59cd47a08c88b7f5d285f0c1bfe)